chapters/02-theory.Rmd

```diff
@@ -0,0 +1,158 @@
+# Theory {#mathematicalTheory}
+
+## Exact Kernel Density Estimation
+
+An exact Kernel Density Estimation can be calculated as
+
+\begin{equation}
+\widehat{f}_h(x) = \frac{1}{nh} \sum_{k=1}^n K\Big(\frac{x-x_k}{h}\Big)
+(\#eq:kde)
+\end{equation}
+
+where $K(x)$ is called the kernel function, which defines the range and size of influence of a single data point over the estimation. Most typically a simple Gaussian distribution is used as kernel function.
```

Comment:
[7] specify _each_ symbol used in the equation (applicable for all following equations)